src/components/dspPricing/PricingPlansDialog.tsx

```diff
@@ -0,0 +1,95 @@
+import { Box, IconButton, Stack } from "@mui/material";
+import { Dialog, Typography } from "elements";
+import { JSX } from "react";
+import LeafFill from "assets/images/LeafFillIcon";
+import theme from "theme";
+import SeedlingFillIcon from "assets/images/SeedlingFillIcon";
+import StarFillIcon from "assets/images/StarFillIcon";
+import CloseIcon from "@mui/icons-material/Close";
+import PricingPlanOption from "./PricingPlanOption";
+import pricingPlanData from "./pricingPlanData.json";
+
+const ICON_SIZE = "20px";
+
+const PRICING_PLAN_ICON: Record<string, JSX.Element> = {
+  collaborator: <LeafFill sx={ { fontSize: ICON_SIZE } } />,
+  artist: <SeedlingFillIcon sx={ { fontSize: ICON_SIZE } } />,
+  artistPlus: <StarFillIcon sx={ { fontSize: ICON_SIZE } } />,
+};
+
+interface PricingPlansDialogProps {
+  readonly handleClose: () => void;
+  readonly open: boolean;
+}
+
+const PricingPlansDialog = ({ handleClose, open }: PricingPlansDialogProps) => {
+  return (
+    <Dialog onClose={ handleClose } open={ open } fullScreen sx={ { m: 5 } }>
```

Comment:
I'm going to ask this to Design/PM as well to get consistent nomenclature across our frontend implementations, but what is our definition of `Dialog` vs `Modal` vs whatever `Toast` style pop ups are? The CU task and other references just note this as a "Popup" and I feel we should get it straight across all parties.

It seems like this should be a `Modal` since it takes up the entire screen and doesn't let the user interact with anything else on the page. However, to match the Figma design UI I needed to use Dialog with some customizations.